core/trino-main/src/main/java/io/trino/server/security/ServerSecurityModule.java

```diff
@@ -73,7 +73,10 @@ protected void setup(Binder binder)
             configBinder(certificateBinder).bindConfig(CertificateConfig.class);
         }));
         installAuthenticator("kerberos", KerberosAuthenticator.class, KerberosConfig.class);
-        installAuthenticator("password", PasswordAuthenticator.class, PasswordAuthenticatorConfig.class);
+        install(authenticatorModule("password", PasswordAuthenticator.class, unused -> {
```

Comment:
I think that `unused` should be actually `used`. The contract of `Module` is that it gives you a `Binder` that should be used.

Reply:
That is an interesting point. It is used indeed, just from a different scope.